{"rendered": {"message": {"raw": "Python 3 bug fix caused by sys.stdout being None in some cases", "markup": "markdown", "html": "<p>Python 3 bug fix caused by sys.stdout being None in some cases</p>", "type": "rendered"}}, "hash": "ae4884e19bb5292b76e385309ca80c7a359bc166", "repository": {"links": {"self": {"href": "data/repositories/lkohfahl/blacs.json"}, "html": {"href": "#!/lkohfahl/blacs"}, "avatar": {"href": "data/bytebucket.org/ravatar/{a6136ac4-0571-4424-bae8-f63c7fe7f2c4}ts=python"}}, "type": "repository", "name": "BLACS", "full_name": "lkohfahl/blacs", "uuid": "{a6136ac4-0571-4424-bae8-f63c7fe7f2c4}"}, "links": {"self": {"href": "data/repositories/lkohfahl/blacs/commit/ae4884e19bb5292b76e385309ca80c7a359bc166.json"}, "comments": {"href": "data/repositories/lkohfahl/blacs/commit/ae4884e19bb5292b76e385309ca80c7a359bc166/comments_page=1.json"}, "patch": {"href": "https://api.bitbucket.org/2.0/repositories/lkohfahl/blacs/patch/ae4884e19bb5292b76e385309ca80c7a359bc166"}, "html": {"href": "#!/lkohfahl/blacs/commits/ae4884e19bb5292b76e385309ca80c7a359bc166"}, "diff": {"href": "https://api.bitbucket.org/2.0/repositories/lkohfahl/blacs/diff/ae4884e19bb5292b76e385309ca80c7a359bc166"}, "approve": {"href": "https://api.bitbucket.org/2.0/repositories/lkohfahl/blacs/commit/ae4884e19bb5292b76e385309ca80c7a359bc166/approve"}, "statuses": {"href": "data/repositories/lkohfahl/blacs/commit/ae4884e19bb5292b76e385309ca80c7a359bc166/statuses_page=1.json"}}, "author": {"raw": "PhyNerd <janwerkmann@gmail.com>", "type": "author", "user": {"display_name": "Jan Werkmann", "uuid": "{44c4905c-2b90-4045-a5f1-652b8e228626}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B44c4905c-2b90-4045-a5f1-652b8e228626%7D"}, "html": {"href": "https://bitbucket.org/%7B44c4905c-2b90-4045-a5f1-652b8e228626%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:a70cc9cf-684e-4849-a61a-9ade4d7218b5/07e5095a-4741-4dc0-a462-9c7d455f961d/128"}}, "nickname": "PhyNerd", "type": "user", "account_id": "557058:a70cc9cf-684e-4849-a61a-9ade4d7218b5"}}, "summary": {"raw": "Python 3 bug fix caused by sys.stdout being None in some cases", "markup": "markdown", "html": "<p>Python 3 bug fix caused by sys.stdout being None in some cases</p>", "type": "rendered"}, "participants": [], "parents": [{"hash": "ae4809380c8e88fa741b8877196aaf9ae77a748b", "type": "commit", "links": {"self": {"href": "data/repositories/lkohfahl/blacs/commit/ae4809380c8e88fa741b8877196aaf9ae77a748b.json"}, "html": {"href": "#!/lkohfahl/blacs/commits/ae4809380c8e88fa741b8877196aaf9ae77a748b"}}}], "date": "2017-11-17T16:05:00+00:00", "message": "Python 3 bug fix caused by sys.stdout being None in some cases", "type": "commit", "git_hash": "9f18f022c0162fb04f1407c1e18350fb66da1234", "tags": null, "branches": "Python3"}